src/components/filter/expert/expert-filter-constants.ts

```diff
@@ -1202,6 +1225,23 @@ export const fields: Record<string, Field[]> = {
         FIELDS_OPTIONS.SUBSTATION_PROPERTY,
         FIELDS_OPTIONS.VOLTAGE_LEVEL_PROPERTY,
     ],
+    DANGLING_LINE: [
+        FIELDS_OPTIONS.ID,
+        FIELDS_OPTIONS.NAME,
+        FIELDS_OPTIONS.COUNTRY,
+        FIELDS_OPTIONS.VOLTAGE_LEVEL_ID,
+        FIELDS_OPTIONS.NOMINAL_VOLTAGE,
```

Comment:
May be missing CONNECTED which is implemented in the backend

Reply:
I missed it